gen.py

```diff
@@ -55,7 +55,8 @@ def usage():
         ("SERVER36", "artifacts/OMERO.server-@VERSION@-ice36-@BUILD@.zip"),
         ("PYTHON36", "artifacts/OMERO.py-@VERSION@-ice36-@BUILD@.zip"),
         ("JAVA36", "artifacts/OMERO.java-@VERSION@-ice36-@BUILD@.zip"),
-        ("DOCS", "artifacts/OMERO.docs-@VERSION@-ice36-@BUILD@.zip"),
+        ("DOCS", "artifacts/OMERO.apidoc-@VERSION@-ice36-@BUILD@.zip"),
+        ("DOC", "artifacts/OMERO-doc-@VERSION@.zip"),
```

Comment:
Hmm, I hadn't noticed this was inconsistent, shall i change the docs PRs to make it OMERO.doc-@version@?